nuxeo-java-client-test/src/test/resources/test-core-types-contrib.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0"?>
+<component name="org.nuxeo.java.client.test.doctypes">
+
+  <require>org.nuxeo.ecm.core.CoreExtensions</require>
+
+  <extension target="org.nuxeo.ecm.core.schema.TypeService" point="schema">
+    <schema name="birthday" src="schemas/birthday.xsd" prefix="bd" />
+  </extension>
+
+  <extension target="org.nuxeo.ecm.core.schema.TypeService" point="doctype">
+    <doctype name="Birthday" extends="Document">
```

Comment:
I would prefer to not have test contributions for stuff that is already on the platform, here a document type with a String field and a Date field.
Those contributions will make the test migration (to test a remote Nuxeo Server) less easy.